### PR TITLE
KV: Add enabledForDiskEncryption and enabledForTemplateDeployment

### PR DESCRIPTION
**Pull request by seanbamsft**
Add enabledForDiskEncryption and enabledForTemplateDeployment flags for editing the Azure Key Vault access policy.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@seanbamsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (seanba). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: setup/azurecmdfiles.wxi [hunk 1]
```diff
@@ -1283,15 +1283,6 @@
                         <Component Id="cmp1C7EFB238F49BC10B88911B4DD7CD760" Guid="*">
                             <File Id="filC52B079BABFAD11D6B89732C6569172A" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.KeyVault\Microsoft.Azure.ResourceManager.dll" />
                         </Component>
-                        <Component Id="cmpF469AE6EA612B0399957FAF85083221B" Guid="*">
```
Comment (markcowl) [inline setup/azurecmdfiles.wxi]: @seanbamsft Are these dependencies not needed by storage?

Comment (seanbamsft) [inline setup/azurecmdfiles.wxi]: Key Vault cmdlets do not use storage. The references were probably left over from the initial copy/paste to create the project.

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/323/

**Pull request merged**